forge-ai/src/main/java/forge/ai/AiBlockController.java

```diff
@@ -204,7 +205,7 @@ private void makeGoodBlocks(final Combat combat) {
                     }
                     blocker = ComputerUtilCard.getWorstCreatureAI(killingBlockers);
                 // 2.Blockers that won't get destroyed
-                } else if (!attacker.hasKeyword("You may have CARDNAME assign its combat damage as though it weren't blocked.")
+                } else if (StaticAbilityAssignCombatDamageAsUnblocked.assignCombatDamageAsUnblocked(attacker)
```

Comment:
missing !

Reply:
Good eye

I left that there to test you... heh